test/cri-containerd/runpodsandbox_test.go

```diff
@@ -252,6 +252,25 @@ func Test_RunPodSandbox_PhysicalMemory_WCOW_Hypervisor(t *testing.T) {
 	runPodSandboxTest(t, request)
 }
 
+func Test_RunPodSandbox_FullyPhysicallyBacked_WCOW_Hypervisor(t *testing.T) {
+	requireFeatures(t, featureWCOWHypervisor)
```

Comment:
Kevin would like if we leave a blank line after our requireFeatures calls by convention.